feature-libs/organization/administration/components/shared/list/list.component.html

```diff
@@ -15,6 +15,7 @@ <h3>
                 appendToBody: true,
                 displayCloseButton: true
               }"
+              [attr.aria-label]="'organization.hint' | cxTranslate"
```

Comment:
Is "hint" really adequate for an aria-label?

Reply:
I was deciding between hint vs. information but popup content is coming from the hint field so I decided to use hint...

Reply:
UX told me to use "information" so I will update